### DISCUSSION TITLE
Can HierarchicalMenu display > 1 level without a user selection?

### DISCUSSION THREAD

**Discussion by avremel** [algolia/instantsearch#5671] [Q&A] [answered]
When using `useHierarchicalMenu` is it possible to render level N + 1 categories without the user making a selection, like in this screenshot:

<img width="201" alt="Screen Shot 2023-06-12 at 10 17 26 AM" src="https://github.com/algolia/instantsearch/assets/10588097/94f4c14c-cc51-4f67-8aa0-914fb59416f6">

**Comment by Haroenv** [comment, marked as answer]
Unfortunately, the design for the hierarchical menu has joined the concepts of refining and displaying the levels, as each level requires its own query, which needs to know about the preceding levels existing. Without user-input this would result in a waterfall of requests, which couldn't be a good experience.

If you have a somewhat limited number of top-level items, you could recreate the experience with a hardcoded (or dynamic widgets) set of hierarchical menus on the first level, each starting on the second level.
